src/menus.c

```diff
@@ -95,21 +95,24 @@ const s8 D_800F2B50[] = {0, 1, 2, 3, 3, 0, 0, 0};
 // set to D_8018EDF3, then that sets gPlayerCountSelection1
 const s8 D_800F2B58[] = {1, 2, 2, 3, 4, 0, 0, 0};
 
-const s8 D_800F2B60[5][3] = {
-    {1, 2, 1},
-    {1, 2, 1},
-    {0, 2, 2},
-    {0, 2, 0}, 
-    {0, 2, 0}
+// limit for each colone in one-two-three-four players selection
```

Comment:
Do you mean column? And could you please capitalize the first letter of the sentence:
`Limit for each column...`